src/frontend/src/components/SearchPage/SearchResults.jsx

```diff
@@ -61,7 +66,9 @@ const SearchResults = ({ text, filter }) => {
       {data && data.length ? (
         <Timeline pages={data} nextPage={() => setSize(size + 1)} />
       ) : (
-        <h1>No search results</h1>
+        <h1 className={classes.noResults}>
+          <img src={NotFound} alt="Not Found" />
```

Comment:
would a `div` tag more appropriate here? `h1` tends to be the title of the page.

Reply:
Agree, I would `<div>` here as well